src/functions/apiFunctions.tsx

```diff
@@ -60,20 +97,36 @@ export async function getTotalGenres(access_token: string, time_range: string):
   return new Map<string, number>();
 }
 
-export function tallyGenres(data: ArtistData): Map<string, number> {
-  let genreCounts = new Map<string, number>();
-
-  if (data.items) {
-    data.items.forEach((item) => {
-      item.genres.forEach((genre) => {
-        if (genreCounts.has(genre)) {
-          genreCounts.set(genre, genreCounts.get(genre)! + 1);
-        } else {
-          genreCounts.set(genre, 1);
-        }
+export async function getArtistInfo(access_token: string, href: string): Promise<ArtistData> {
+  if (access_token) {
+    try {
+      const response = await fetch("api/data/artist", {
+        method: "POST",
+        mode: "no-cors",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({ access_token, href }),
       });
+      const artistData = (await response.json()) as ArtistData;
+      if (artistData) {
+        return artistData;
+      }
+    } catch (error) {
+      console.log(error);
```

Comment:
Keep it consistent and use console.error here and below

Reply:
changed all .logs to .error